Doc/library/resource.rst

```diff
@@ -76,6 +76,8 @@ this module for those platforms.
    ``setrlimit`` may also raise :exc:`error` if the underlying system call
    fails.
 
+   VxWorks only support setting RLIMIT_NOFILE and idtype must be P_PID.
```

Comment:
"supports" with a S, no? (I'm not sure, English is not my first language!) Please use :data: to format properly the RLIMIT_NOFILE constant.

```suggestion
   VxWorks only supports setting :data:`RLIMIT_NOFILE` and idtype must be P_PID.
```

I don't understand "idtype must be P_PID": what does it mean? Is it something specific to VxWorks?

Reply:
Yes, it's VxWorks specific requires.

Reply:
I remove that statements for python user doesn't need to care about it.